### ISSUE TITLE
Experiments collection

### ISSUE THREAD

**Issue by CJ-Wright** [regro/regolith#128]
It would be great to have a collection for experiments (including computational experiments).

This collection would contain information about:
- the lead experimenter for the experiments
- who is performing the experiments
- the project(s) included in the experiments
- The date(s) of the experiments
- Any samples/data associated with the experiments (not required, but helpful esp. for physical samples)
- More things

@sbillinge thoughts?

**Comment by sbillinge** [comment]
yes.  I think I started this and called it beamtimes.yml, but experiments
is more general I guess.

Looking at that file, it is not properly filled out.  We should think what
needs to be in there.  I guess that you would like to automatically build
the beamline sample spreadsheet or sthg, but this might be too much?
Better might be to have an ID for the beamtime and link to that in the
sampleDB.

I agree that the full list of experimenters would be good.

S

**Comment by CJ-Wright** [comment]
Building the sample spreadsheet would be good. If we could access the PASS system then we could directly issue the PASS forms and all. My hope is that this would be a rather free from collection, since a) we can have a subschema for beamtimes (which maybe have a special key for the type of experiment) and b) it may be very difficult to know ahead of time all the potential interesting things about the experiment.

**Comment by sbillinge** [comment]
a problem I think is that the sample spreadsheet is a bit of a moving
target.  Is it even finalized when everyone heads off to the beamline, or
are things added even after getting there?

But the workflow should probably be:
1) students enter samples into the sampleDB
2) lead experimenter runs `regolith experiment`
3) `experiment` asks for information about the experiment like dates and
things, but also asks lead experimenter to select experiments from a list
of samples it finds in sampleDB

sthg like that.

S

**Comment by CJ-Wright** [comment]
Everything has to be entered into PASS for chemical safety at some point right? There is also nothing stopping us from amending documents in the collection after they have been added. One could consider writing the experiment information as soon as the time is allocated and then having people update it as more samples/experimenters/projects are added to the experiment.

I guess the main question for me is what do we want to get out of this collection?
1. The PASS sample safety form?
1. A report about all the beamtime's activity?
1. A place to store common data for each experimenter's report(s) on their individual samples?

How do we separate a campaign of experiments (potentially spanning multiple beamtimes or synthesis runs) from the experiment itself? (Do we care about this separation (should we auto-generate beamtime proposals?)?)
